libs/langchain/tests/unit_tests/chat_models/test_anyscale.py

```diff
@@ -0,0 +1,54 @@
+"""Test Anyscale chat model."""
+from typing import List
+
+import pytest
+from pytest import CaptureFixture, MonkeyPatch
+
+from langchain.chat_models.anyscale import DEFAULT_MODEL, ChatAnyscale
+from langchain.pydantic_v1 import SecretStr
+
+
+@pytest.fixture(autouse=True)
+def prevent_anyscale_api_request(monkeypatch: MonkeyPatch) -> None:
+    def mocked_get_available_models(arg1: None, arg2: None) -> List[str]:
+        return [DEFAULT_MODEL]
+
+    monkeypatch.setattr(
+        ChatAnyscale,
+        "get_available_models",
+        mocked_get_available_models,
+    )
+
+
+@pytest.mark.requires("openai")
```

Comment:
one more thing that i noticed -- anyscale doesn't use openai under the hood, so we don't need to use `requirs("openai")`

Reply:
oh sorry totally wrong -- yes it does use openai, so this is fine